must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -100,6 +104,8 @@ done
 # NOTE: This is a temporary fix for collecting the storagecluster as we are not able to collect the storagecluster using the inspect command
 { oc get storageclusters -n ${INSTALL_NAMESPACE} -o yaml; } > "$BASE_COLLECTION_PATH/namespaces/${INSTALL_NAMESPACE}/oc_output/storagecluster.yaml" 2>&1
 { oc get storagesystem -n ${INSTALL_NAMESPACE} -o yaml; } > "$BASE_COLLECTION_PATH/namespaces/${INSTALL_NAMESPACE}/oc_output/storagesystem.yaml" 2>&1
+{ oc get storageconsumer -n ${INSTALL_NAMESPACE} -o yaml; } > "$BASE_COLLECTION_PATH/namespaces/${INSTALL_NAMESPACE}/oc_output/storageconsumer.yaml" 2>&1
+{ oc get cephfilesystemsubvolumegroups.ceph.rook.io -n ${INSTALL_NAMESPACE} -o yaml; } > "$BASE_COLLECTION_PATH/namespaces/${INSTALL_NAMESPACE}/oc_output/cephfilesystemsubvolumegroups.ceph.rook.io.yaml" 2>&1
```

Comment:
cephfilesystemsubvolumegroups should be treaded the same as cephblockpool (its again Rook specific CRD) I see this is already collected at [gather_ceph_resources](https://github.com/red-hat-storage/ocs-operator/pull/1577/files#diff-8ef2ba262f2b9ddce504e6326eb01c785d5afdb12679ba29a3ee5f1fa19bf636)

Reply:
yes, will remove this part. in gather_ceph_resources only yaml is getting collected and as mentioned by @nehaberry we also need details for oc get and desc of `cephfilesystemsubvolumegroups.ceph.rook.io`, hence collecting those as well.